### ISSUE TITLE
Autoload does not work

### ISSUE THREAD

**Issue by urizev** [Insure-Tech/InsuranceHub.Client.Php#3]
### How to reproduce

1. Install composer in a clean directory:

```shell
$ php composer.phar require booking-protect/insurance-hub-client
```

2. Create a simple PHP file (index.php) which uses some library class.

```php
<?php

require_once 'vendor/autoload.php';

$api = new BookingProtect\InsuranceHub\Client\ApiClientConfiguration();

var_dump_api();
```

3. Execute script:

```shell
$ php index.php
PHP Fatal error:  Uncaught Error: Class 'BookingProtect\InsuranceHub\Client\ApiClientConfiguration' not found in /var/www/buffer/test/index.php:5
Stack trace:
#0 {main}
  thrown in /var/www/buffer/test/index.php on line 5
```

### Environment
```shell
$ php --version
PHP 7.4.16 (cli) (built: Mar  5 2021 08:37:59) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.16, Copyright (c), by Zend Technologies

$ php composer.phar -V
Composer version 2.1.3 2021-06-09 16:31:20
```

**Comment by saarea** [comment]
Hi,

Thanks for raising this.  We noticed your pull request earlier and are happy to bring your changes in.  We just need to review them.  We're a little short on resource at the moment due to vacations but we will get round to it ASAP.  

Could you please create the pull request against the develop branch?

We very much appreciate your feedback and your improvements.

**Comment by urizev** [comment]
Hi,

I appreciate your words.

I am integrating your platform and it would be great to be able to do that via composer. Since it was buggy I decided to fork the repo and make some refactor.

What I mean is that I didn't just fix the autoload bug but I make quite a few changes in the code. Anyway, I will create the pull request again against your develop branch in case you still are interested.

Regards

**Comment by saarea** [comment]
fixed with release 1.2.0
